### DISCUSSION TITLE
possibly leak of grpc connection

### DISCUSSION THREAD

**Discussion by benoitTK** [fennec-project/podtracer#20] [General]
I suggest to add the following instruction into a defer after a successful Dial

https://github.com/fennec-project/podtracer/blob/cc033f4c2e1c2530171e572f452b1fa63fd7798a/cmd/internal/podtracer/container.go#L151

**Comment by acmenezes** [comment]
Hi @benoitTK , totally appreciate your suggestion. I just opened an issue for that (#22 ). I'll check that out later today. Feel free to review/contribute with anything you want. Most here was done in pretty PoC informal way and is subject to change and be improved. We'll be publishing our regular meeting links and also more instructions on our projects this week. Thanks a lot!
